[16-Feb-2022 17:00:26 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(34): Core\Router->dispatch('csstest')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:02:13 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(34): Core\Router->dispatch('csstest')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:02:16 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(34): Core\Router->dispatch('csstest')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:02:18 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(34): Core\Router->dispatch('csstest')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:02:22 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(34): Core\Router->dispatch('csstest')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:02:36 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(34): Core\Router->dispatch('csstest')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:02:39 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(34): Core\Router->dispatch('csstest')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:02:50 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(34): Core\Router->dispatch('csstest')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:02:58 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(34): Core\Router->dispatch('csstest')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:03:00 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(34): Core\Router->dispatch('csstest')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:03:05 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(34): Core\Router->dispatch('csstest')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:41:50 Europe/Berlin] Uncaught exception: 'ParseError' with message: 'syntax error, unexpected '" class="' (T_CONSTANT_ENCAPSED_STRING), expecting ',' or ';'' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Response.php(18): Core\View::render('Home/index.php', Array)
#1 C:\xampp\htdocs\simpl\App\Controllers\Home.php(18): Core\Response::phpHTMLResponse('Home/index.php', 'homedata', 'homedata')
#2 [internal function]: App\Controllers\Home->indexAction()
#3 C:\xampp\htdocs\simpl\Core\Controller.php(21): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\simpl\Core\Router.php(63): Core\Controller->__call('index', Array)
#5 C:\xampp\htdocs\simpl\public\index.php(35): Core\Router->dispatch('')
#6 {main} 
Trown in : 'C:\xampp\htdocs\simpl\App\Views\PhpHtml\header.php' on line 36
[16-Feb-2022 17:41:50 Europe/Berlin] PHP Parse error:  syntax error, unexpected '" class="' (T_CONSTANT_ENCAPSED_STRING), expecting ',' or ';' in C:\xampp\htdocs\simpl\App\Views\PhpHtml\header.php on line 36
[16-Feb-2022 17:41:50 Europe/Berlin] PHP Fatal error:  Exception thrown without a stack frame in Unknown on line 0
[16-Feb-2022 17:42:59 Europe/Berlin] Uncaught exception: 'ParseError' with message: 'syntax error, unexpected 'text' (T_STRING), expecting ',' or ';'' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Response.php(18): Core\View::render('Home/index.php', Array)
#1 C:\xampp\htdocs\simpl\App\Controllers\Home.php(18): Core\Response::phpHTMLResponse('Home/index.php', 'homedata', 'homedata')
#2 [internal function]: App\Controllers\Home->indexAction()
#3 C:\xampp\htdocs\simpl\Core\Controller.php(21): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\simpl\Core\Router.php(63): Core\Controller->__call('index', Array)
#5 C:\xampp\htdocs\simpl\public\index.php(35): Core\Router->dispatch('')
#6 {main} 
Trown in : 'C:\xampp\htdocs\simpl\App\Views\PhpHtml\header.php' on line 36
[16-Feb-2022 17:42:59 Europe/Berlin] PHP Parse error:  syntax error, unexpected 'text' (T_STRING), expecting ',' or ';' in C:\xampp\htdocs\simpl\App\Views\PhpHtml\header.php on line 36
[16-Feb-2022 17:42:59 Europe/Berlin] PHP Fatal error:  Exception thrown without a stack frame in Unknown on line 0
[16-Feb-2022 17:43:08 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(35): Core\Router->dispatch('Css Text')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:43:09 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(35): Core\Router->dispatch('Home')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:43:10 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(35): Core\Router->dispatch('Css Text')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:43:54 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(35): Core\Router->dispatch('Css Text')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:43:57 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(35): Core\Router->dispatch('Css Text')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:44:17 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(35): Core\Router->dispatch('Css Text')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:44:17 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(35): Core\Router->dispatch('Home')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:44:53 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(35): Core\Router->dispatch('home')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:45:38 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(35): Core\Router->dispatch('home')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:45:44 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(35): Core\Router->dispatch('home')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:45:54 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(35): Core\Router->dispatch('home')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:46:00 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(35): Core\Router->dispatch('home')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:46:07 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(35): Core\Router->dispatch('home')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:46:10 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(35): Core\Router->dispatch('home')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:46:19 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(35): Core\Router->dispatch('home')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:46:20 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(35): Core\Router->dispatch('home')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
[16-Feb-2022 17:46:22 Europe/Berlin] Uncaught exception: 'Exception' with message: 'Not Found' 
Stack trace: '#0 C:\xampp\htdocs\simpl\Core\Router.php(46): Core\ErrorHandler::throwAndLogErr('No route mached', 'Not Found', 404)
#1 C:\xampp\htdocs\simpl\public\index.php(35): Core\Router->dispatch('home')
#2 {main} 
Trown in : 'C:\xampp\htdocs\simpl\Core\ErrorHandler.php' on line 13
